package.json

```diff
@@ -13,7 +13,8 @@
   "pnpm": {
     "overrides": {
       "workerd": "^1.20240729.0",
-      "@remix-run/dev": "npm:@dario-hacking/remix-run-dev@2.9.1-vite-env-4"
+      "@remix-run/dev": "/Users/dario/Repos/my-repos/remix/packages/remix-dev",
+      "vite": "https://pkg.pr.new/vite@main"
```

Comment:
this points to the latest vite version from their main branch

we can use this or wait for their next release and keep using more proper/stable npm releases

Reply:
I just worry that someone else might look at the project and then it could be broken because of a configuration change. I think bumping the version numbers manually and checking for changes might be better.

Reply:
let's just go with `6.0.0-beta.0` 🙂